integration-tests/integration_tests/integration_tests/assertors/text_error_response_assertor.py

```diff
@@ -1,80 +1,37 @@
 """
-Provides assertions over the text response format we get from the MHS when an error is returned from Spine
```

Comment:
The name of this file says `text_error_response_assertos`. Should it be `text` since you changed that to json?

Reply:
changing that to json_error_response_assertor.py